scss/_layout.scss

```diff
@@ -69,25 +83,128 @@ h2 {
         position: relative;
         z-index: 1;
         padding: 1rem;
-        border: 2px solid grey;
     }
 
+    .current-container {
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: cover;
+        height: 400px;
+        border-radius: 10px;
 
-    .favorite-btn {
+        &--day {
+            background-image: url($daytime-bg-pic);
```

Comment:
This time is totally fine, but for the real application or website, we resize the background image and try to small like less than 100kb. You can even use webp instead of jpeg or png for lighter images! You can take a look when you have time :)